### PR TITLE
make slight modifications to the Partners page

### LINKED ISSUE #44: Front-end: Partners Page
# Purpose
There are some minor changes that need to be made to the OSL site.

# Description
Update the Partners Page with changes marked from the Davids in this [doc ](https://docs.google.com/document/d/1O2HuRZnJbe8Wgx3U5zy11OfPvDkGRgMl/edit?usp=sharing&ouid=105775914785337038980&rtpof=true&sd=true)
- [ ] Update logos
- [ ] Add equality [logo](https://drive.google.com/file/d/1L9xPben6LmaOeI-hXZNdoCO2Z8np6lUF/view?usp=sharing) to the existing logos
- [ ] Add a "thank you to Equality Fund" note at the center-bottom of the partners page
- [ ] LGBTQIA+ Aging Project [logo](https://drive.google.com/file/d/1MlH0D-8epU1klRWvo_dDaaUMdVC1FHG3/view?usp=sharing)

# Acceptance Criteria
- [ ] Please attach before and after screenshots of the changes you've made to these pages.

### PR DESCRIPTION
**Pull request by saahilkumar**
### ℹ️ Issue

Closes #44 

### 📝 Description

Briefly list the changes made to the code:

- Removed space so `Friendship Works` became `FriendshipWorks`
- Added three new partners:  The Equality Fund, LGBTQIA+ Aging Project, and Springwell
- Added a lil message at the bottom thanking The Equality Fund for their donation

Screenshots:

**Friendship Works before:**

Screen Shot 2022-08-03 at 1 46 54 PM

**FriendshipWorks after:**

Screen Shot 2022-08-03 at 1 47 02 PM

**Before new partners and footer message:**

Screen Shot 2022-08-03 at 1 47 20 PM

**After new partners and footer message:**

Screen Shot 2022-08-03 at 2 21 18 PM

### ✔️ Verification

Modified the cypress test a lil

### CODE REVIEW

**Comment by saahilkumar** [comment]
Btw, I grabbed the Springwell and LGBTQIA+ Aging Project logos as screenshots so they might be a lil fuzzy. Lmk if there's an image somewhere with higher quality that I should use instead.

File: pages/partners/index.tsx [hunk 1]
```diff
@@ -46,6 +46,24 @@ const partnersData: PartnersListingProps[] = [
     imageHeight: '362px',
     url: 'https://www.lgbtqseniorhousing.org/',
   },
+  {
+    name: 'The Equality Fund',
+    imageSrc: '/ef.jpg',
+    imageHeight: '58px',
```
Comment (yanDavid21) [inline pages/partners/index.tsx:52]: how are you choosing these heights?

File: pages/partners/index.tsx [hunk 2]
```diff
@@ -96,6 +114,13 @@ const Partners: React.FC<{}> = () => {
           ))}
         </Flex>
       </div>
+
+      <div data-cy="partners-footer">
+        <Text mt="32px" textAlign="center">
```
Comment (yanDavid21) [inline pages/partners/index.tsx:119]: not the biggest fan of using px for css, would prefer em, rm or some other constant/multiple

File: pages/partners/index.tsx [hunk 3]
```diff
@@ -46,6 +46,24 @@ const partnersData: PartnersListingProps[] = [
...
   },
+  {
+    name: 'The Equality Fund',
+    imageSrc: '/ef.jpg',
+    imageHeight: '58px',
+    url: 'https://www.tbf.org/nonprofits/grant-making-initiatives/equality-fund',
+  },
+  {
+    name: 'LGBTQIA+ Aging Project',
+    imageSrc: '/lap.png',
```
Comment (yanDavid21) [inline pages/partners/index.tsx:57]: what do loading times look for these images? should we host these on s3 to reduce time for meaningful content?

**Review by yanDavid21 — DISMISSED** [review]
LGTM (minor optimizations commented but non-blocking)

**Comment by adriannaliu** [comment]
> Btw, I grabbed the Springwell and LGBTQIA+ Aging Project logos as screenshots so they might be a lil fuzzy. Lmk if there's an image somewhere with higher quality that I should use instead.

i updated the ticket with the LGBTQIA+ Aging Project logo they sent us. i'll get you the SpringWell logo sooooon

File: pages/partners/index.tsx (hunk 1 above)
Comment (saahilkumar) [reply pages/partners/index.tsx:52]: Varun made the width a constant 360px so the height is just scaled down to match the original dimensions of the logo

File: pages/partners/index.tsx (hunk 2 above)
Comment (saahilkumar) [reply pages/partners/index.tsx:119]: not the biggest fan of u

also we had this discussion a while ago i'm p sure and Varun mentioned this:
https://medium.com/hackernoon/rems-and-ems-and-why-you-probably-dont-need-them-664b9ce1e09f

File: pages/partners/index.tsx (hunk 3 above)
Comment (saahilkumar) [reply pages/partners/index.tsx:57]: yea it's a lil slow, i'm p sure we also talked about s3 before. Idk what the decision was but that's out of scope of this ticket

**Comment by saahilkumar** [comment]
Added high def logos

**Review by adriannaliu — APPROVED** [review]
wonderful!

**Review by thejamielin — APPROVED** [review]
amazing saahil!

**Pull request merged**